cram_3d_world/cram_bullet_reasoning/src/items.lisp

```diff
@@ -171,6 +171,85 @@ The name in the list is a keyword that is created by lispifying the filename."
                                                                       mass)
   (add-object world :mesh name pose :mass mass :mesh :mug))
 
+(defun make-rectangle-shape (lenght width height)
```

Comment:
`make-rectangle-shape` is a weird name for this function. It's not really a rectangle that you are creating.

I think, it would probably be best to just put all these parts of the basket into one function. So combine `make-rectangle-shape`, `make-handle-shape` and `make-basket-shape` together and put it into `add-object :basket`.
I don't see us using any of these functions for any other object at this moment. For now, it is only confusing what this `make-rectangle-shape` is used for. But when you know that this is a part of a basket, it is much clearer.